spec/requests/corporate_information_page_spec.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.describe "Corporate Information Page" do
```

Comment:
We might need an audit trail message for the system/request tests?

Reply:
Good spot thanks @KludgeKML , I've updated in https://github.com/alphagov/frontend/pull/4608/commits/c21d7fa549aa9856b2c15ebe89dfd7c481c9587a